### PR TITLE
Standardize all 3 real world example tutorials

### PR DESCRIPTION
**Pull request by jfomhover**
## Purpose

Aligning all 3 tutorials with a similar structure and wording. Validate instructions with an outside eye + bug bash.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Review by majercakdavid — APPROVED** [review]

File: docs/real-world-examples/ccfraud.md [hunk 1]
```diff
@@ -1,49 +1,87 @@
...
+   ```
+
+    :star: you can simplify this command by entering your workspace details in the file `config.yaml` in this same directory.
+
+## Beyond the experiment
+
+This sample experiment provides multiple models you can try.
 
-### Model choice
 Please update [**config**](../../examples/pipelines/ccfraud/config.yaml), field `model_name` in the `training_parameters` section, to reflect desired model to be trained, options include: SimpleLinear, SimpleLSTM, SimpleVAE
```
Comment (thomasp-ms) [inline docs/real-world-examples/ccfraud.md:83]: This sentence is pretty hard to parse. It could use a couple articles, being split after "trained", and a dot at the end.

**Review by thomasp-ms — CHANGES_REQUESTED** [review]
Looks pretty good. I had a few comments. Some of them I believe you will want to implement, while some others might be a bit subjective so up to you which ones

File: docs/real-world-examples/ccfraud.md (hunk 1 above)
Comment (jfomhover) [reply docs/real-world-examples/ccfraud.md]: Sure, made another proposal.

File: docs/real-world-examples/pneumonia.md [hunk 2]
```diff
@@ -1,53 +1,78 @@
...
+3. Click "Create" button in the top. Lookup currently logged in user (using user id or an email), select it and press "Next". 
+
+4. Press "Next" and "Create" in the next screens.
+
+    We are now able to create a secret in the keyvault.
+
+5. Open the "Secrets" tab. Create two plain text secrets:
+    
+    - **kaggleusername** - specifies your kaggle user name
+    - **kagglekey** - this is API key that can be obtained from your profile on the kaggle
```
[the remark this answers is not in the document]
Comment (jfomhover) [reply docs/real-world-examples/pneumonia.md]: Modified it to read better.

File: docs/real-world-examples/pneumonia.md [hunk 3]
```diff
@@ -1,53 +1,78 @@
...
+python -m pip install -r ./examples/pipelines/requirements.txt
+```
+
+## Provision an FL sandbox workspace
+
+To run this example, you will need to provision an AzureML workspace ready for Federated Learning. We strongly recommend you use the setup provided in the repository [quickstart](../quickstart.md). We will use the same names for the computes and datastores created by default during this quickstart.
+
+:notebook: take note of your workspace name, resource group and subscription id. You will need them to submit the experiment.
+
+## Add your kaggle credentials to the workspace keyvault
```
[the remark this answers is not in the document]
Comment (jfomhover) [reply docs/real-world-examples/pneumonia.md]: Easy fix, Done!

**Review by ShizeSu06 — APPROVED** [review]
LGTM, thanks Jeff for the great efforts!

**Comment by jfomhover** [comment]
@thomasp-ms as you're not available for now, but I took into account your comments, I propose to skip your request for change for now, if there's anything left I can revisit before we merge release 04 in the main branch. Thanks!
